pom.xml

```diff
@@ -1339,5 +1339,27 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>disableWizard</id>
+      <activation>
+        <property>
+          <name>!enableWizard</name>
+        </property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.jenkins-ci.tools</groupId>
+            <artifactId>maven-hpi-plugin</artifactId>
+            <configuration>
+              <systemProperties>
+                <name>hudson.Main.development</name>
+                <value>true</value>
```

Comment:
ugh! overly complicated!

```
diff --git a/pom.xml b/pom.xml
index 3378555..3a67751 100644
--- a/pom.xml
+++ b/pom.xml
@@ -88,6 +88,7 @@
     <!-- For jgit-scm-provider profile -->
     <git.provider>git</git.provider>
     <maven-scm.version>1.9.5</maven-scm.version>
+    <hudson.Main.development>true</hudson.Main.development>
   </properties>
 
   <prerequisites>
@@ -612,6 +613,12 @@
           <showDeprecation>true</showDeprecation>
           <contextPath>/jenkins</contextPath>
           <!-- TODO specify ${java.level} when JENKINS-20679 implemented -->
+          <systemProperties>
+            <systemProperty>
+              <name>hudson.Main.development</name>
+              <value>${hudson.Main.development}</name>
+            </systemProperty>
+          </systemProperties>
         </configuration>
       </plugin>
       <plugin>
```

would seem much simpler and far more The Maven Way™